cclib/parser/nwchemparser.py

```diff
@@ -93,17 +104,24 @@ def extract(self, inputfile, line):
             self.set_attribute('natom', natom)
 
         if line.strip() == "NWChem Geometry Optimization":
-            self.skip_lines(inputfile, ['d', 'b', 'b', 'b', 'b', 'title', 'b', 'b'])
-            line = next(inputfile)
+            try:

```

Comment:
Rather than be dogmatic and say "we must always use `skip_lines` when trying to reach some new text" or something similar, I'd prefer whatever needs to be rewritten so that the `assert` isn't duplicated.